src/components/Navigation.js

```diff
@@ -74,8 +74,10 @@ class Navigation extends React.Component {
                   Tokens
                 </span>
                 <div className="dropdown-menu" aria-labelledby="navbarDropdown">
-                  <ConditionalNavigation to="/tokens" label="Token list" featureToggle={`${UNLEASH_TOKENS_BASE_FEATURE_FLAG}.rollout`} />
-                  <ConditionalNavigation to="/token_balances" label="Token balances" featureToggle={`${UNLEASH_TOKEN_BALANCES_FEATURE_FLAG}.rollout`} />
+                  <ul>
+                    <ConditionalNavigation to="/tokens" label="Token list" featureToggle={`${UNLEASH_TOKENS_BASE_FEATURE_FLAG}.rollout`} />
+                    <ConditionalNavigation to="/token_balances" label="Token balances" featureToggle={`${UNLEASH_TOKEN_BALANCES_FEATURE_FLAG}.rollout`} />
+                  </ul>
```

Comment:
`ConditionalNavigation` had a couple of `<li>` items which were causing trouble since they are inside another `<li>`.
Wrapping the components in an `<ul>` seemed to solve the issue.